compiler/compiler/src/main/java/org/robovm/compiler/target/ios/SDK.java

```diff
@@ -97,10 +113,66 @@ private static String toString(NSObject o) {
     }
     
     private static List<SDK> listSDKs(String platform) {
+        List<SDK> allSdks = new ArrayList<>();
+
+        allSdks.addAll(listOldFileFormatSdks(platform));
+
+        allSdks.addAll(listNewFileFormatSdks());
+
+        return allSdks;
+    }
+
+    private static Collection<? extends SDK> listNewFileFormatSdks() {
+        File sdksDir = new File(SDK_XCODE_8_3_AND_UP_INSTALL_LOCATION);
+        if (!sdksDir.isDirectory()) {
+            return emptyList();
+        }
+
+        List<SDK> sdks = new ArrayList<>();
+
+        for (File sdkRoot : notNull(sdksDir.listFiles())) {
+            if (sdkRoot.getName().matches(IOS_SIMULATOR_MAJOR_MINOR_VERSION_REGEX)) {
+                sdks.add(createNewFileFormatSdk(sdkRoot));
+            }
+        }
+
+        return sdks;
+    }
+
+    /**
+     * New directory format SDKs, as downloaded by Xcode version >= 8.3.
+     * <p>Fills only some of the data of the SDK, but enough to use it as Simulator.</p>
+     */
+    static SDK createNewFileFormatSdk(File sdkRootDir) {
+
+        Matcher matcher = IOS_SIMULATOR_MAJOR_MINOR_VERSION_REGEX_PATTERN.matcher(sdkRootDir.getName());
+        Validate.isTrue(matcher.find());
+
+        String majorVersion = matcher.group(1);
+        String minorVersion = matcher.group(2);
+
+        String name = sdkRootDir.getName();
+
+        SDK sdk = new SDK();
+        sdk.major = Integer.valueOf(majorVersion);
+        sdk.minor = Integer.valueOf(minorVersion);
+        sdk.minimalDisplayName = name;
+        sdk.displayName = name;
+        sdk.canonicalName = name;
+        sdk.version = majorVersion + "." + minorVersion;
+        sdk.root = sdkRootDir;
+
+        return sdk;
+    }
+
+    /**
+     * Old style SDKs, as downloaded by Xcode version < 8.3.
+     */
```

Comment:
Likewise, these are not old SDKs, but bundled SDKs with xcode. XCode 9 still uses this for its bundled simulator and device SDKs.